content/tutorial/tutorial.md

```diff
@@ -213,60 +213,60 @@ class Square extends React.Component {
 
-### Making an Interactive Component {#making-an-interactive-component}
+### Tworzenie interaktywnego komponentu {#making-an-interactive-component}
 
-Let's fill the Square component with an "X" when we click it. 
-First, change the button tag that is returned from the Square component's `render()` function to this:
+Niech teraz komponent `Square` wypełnia się literą "X" po kliknięciu w niego.
+Najpierw dokonaj zmian w elemencie `<button />` zwracanym przez `Square` w metodzie `render()`:
 
 ```javascript{4}
 class Square extends React.Component {
   render() {
     return (
-      <button className="square" onClick={function() { alert('click'); }}>
+      <button className="square" onClick={function() { alert('klik'); }}>
         {this.props.value}
       </button>
     );
   }
 }
 ```
 
-If we click on a Square now, we should get an alert in our browser.
+Teraz gdy klikniemy na polu, przeglądarka powinna wyświetlić wiadomość w oknie dialogowym.
 
->Note
+>Uwaga
 >
->To save typing and avoid the [confusing behavior of `this`](https://yehudakatz.com/2011/08/11/understanding-javascript-function-invocation-and-this/), we will use the [arrow function syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) for event handlers here and further below:
+>Żeby oszczędzić sobie pisania i zapobiec [mylącemu zachowaniu `this`](https://yehudakatz.com/2011/08/11/understanding-javascript-function-invocation-and-this/), od teraz obsługę zdarzeń będziemy zapisywać za pomocą [funkcji strzałkowych](https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Functions/Funkcje_strzalkowe) (ang. *arrow functions*):
 >
 >```javascript{4}
 >class Square extends React.Component {
 >  render() {
 >    return (
->      <button className="square" onClick={() => alert('click')}>
+>      <button className="square" onClick={() => alert('klik')}>
 >        {this.props.value}
 >      </button>
 >    );
 >  }
 >}
 >```
 >
->Notice how with `onClick={() => alert('click')}`, we're passing *a function* as the `onClick` prop. It only fires after a click. Forgetting `() =>` and writing `onClick={alert('click')}` is a common mistake, and would fire the alert every time the component re-renders.
+>Zauważ, że za pomocą `onClick={() => alert('klik')}` pod atrybutem `onClick` przekazujemy *funkcję*. Zostanie ona wywołana dopiero po kliknięciu w przycisk. Częstym błędem jest zapominanie o `() =>` i pisanie `onClick={alert('klik')}`, co powoduje wyświetlenie wiadomości w momencie renderowania komponentu.
 
-As a next step, we want the Square component to "remember" that it got clicked, and fill it with an "X" mark. To "remember" things, components use **state**.
+W następnym kroku sprawimy, by komponent `Square` "pamiętał", że został kliknięty, i wyświetlał literę "X". Do "pamiętania" rzeczy komponenty używają **stanu**.
```

Comment:
Za pierwszym przeczytaniem zastanawiałem się, co to są `rzeczy komponenty` i miałem wrażenie, że to literówka (`rzeczy komponentu`), więc zmiana struktury wyeliminuje to ryzyko.

```suggestion
W następnym kroku sprawimy, by komponent `Square` "pamiętał", że został kliknięty, i wyświetlał literę "X". Komponenty do "pamiętania" rzeczy używają **stanu**.
```